docs/resources/integration_zendesk.md

```diff
@@ -0,0 +1,76 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "mondoo_integration_zendesk Resource - terraform-provider-mondoo"
+subcategory: ""
+description: |-
+  Zendesk integration to keep track of security tasks and add Zendesk tickets directly from within the Mondoo Console.
+---
+
+# mondoo_integration_zendesk (Resource)
+
+Zendesk integration to keep track of security tasks and add Zendesk tickets directly from within the Mondoo Console.
+
+## Example Usage
+
+```terraform
+variable "zendesk_token" {
+  description = "The GitHub Token"
+  type        = string
+  sensitive   = true
+}
+
+provider "mondoo" {
+  space = "hungry-poet-123456"
+}
+
+# Setup the zendesk integration
+resource "mondoo_integration_zendesk" "zendesk_integration" {
+  name      = "My Zendesk Integration"
+  subdomain = "your-subdomain"
+  email     = "zendeskowner@email.com"
+  api_token = var.zendesk_token
```

Comment:
Credentials needs to become a nested object to align with the other integrations for secrets:

```
 credentials = {
    api_token = var.zendesk_token
  }
  ```